source/adapter.screenshare.js

```diff
@@ -141,6 +141,10 @@
 
     getUserMedia = navigator.getUserMedia;
 
+  } else if (window.webrtcDetectedBrowser === 'edge') {
```

Comment:
I think you should use
else if (navigator.mediaDevices && navigator.userAgent.match(
    /Edge\/(\d+).(\d+)$/)) {
to keep the same system as the main adapter and not assume this variable was set.